app/com/linkedin/drelephant/exceptions/util/ExceptionInfo.java

```diff
@@ -48,23 +66,81 @@ public void setExceptionID(int exceptionID) {
     this.exceptionID = exceptionID;
   }
 
+  public Integer getWeightOfException() {
```

Comment:
nit: Using lombok annotations will be helpful for Getter/Setter boilerplate code.